oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurer.java

```diff
@@ -384,6 +411,18 @@ private static void validateProviderSettings(ProviderSettings providerSettings)
 		return jwtCustomizer;
 	}
 
+	private static <B extends HttpSecurityBuilder<B>> UserInfoClaimsMapper getUserInfoClaimsMapper(B builder) {
```

Comment:
It looks like when resolving conflicts, the code that configures the filter got dropped. See note on review.

After you rebase on `main`, this should be moved to `OAuth2ConfigurerUtils`.